### PR TITLE
feat (cmx): replicated cluster upgrade command

### PR DESCRIPTION
**Pull request by emosbaugh**
Screenshot 2023-09-11 at 10 12 49 AM

Screenshot 2023-09-11 at 10 13 21 AM

### CODE REVIEW

File: docs/reference/replicated-cli-cluster-kubeconfig.md [hunk 1]
```diff
@@ -17,11 +17,6 @@ replicated cluster kubeconfig [flags]
     <th width="50%">Description</th>
   </tr>
   <Help/>
-  <tr>
-    <td>--id</td>
-    <td>string</td>
-    <td>The ID of the cluster to download credentials for, when the a name is not specified.</td>
-  </tr>
```
Comment (jdewinne) [inline docs/reference/replicated-cli-cluster-kubeconfig.md:24]: `replicated cluster kubeconfig` does have an `--id` flag at the moment. Not sure if it should be there.

**Review by jdewinne — APPROVED** [review]
Added a comment for `cluster kubeconfig`.

File: docs/reference/replicated-cli-cluster-kubeconfig.md (hunk 1 above)
Comment (emosbaugh) [reply docs/reference/replicated-cli-cluster-kubeconfig.md]: i've added this back

**Review by jdewinne — APPROVED** [review]

**Review by paigecalvert — APPROVED** [review]
thanks @emosbaugh and @jdewinne !

**Pull request merged**